docs/developer-guide.md

```diff
@@ -22,6 +23,18 @@
 1. [Prepare the CI to run from your fork](ci-cd.md) (optional)
     - Required to run integration testing on forks of this repo.
 
+### Git leaks detection
+Since the repository contains a secret information in form of encrypted
+Ansible Vault there is high chance that developer may push a commit with
+decrypted secrets by mistake. To avoid this problem we recommend
+to use `Gitleaks` tool that prevent you from commit secret code into git history.
+
+The repository is already pre-configured but each developer has to make final
+config changes in his/her environment.
+
+Follow the [documentation](https://github.com/gitleaks/gitleaks#pre-commit) to
+configure Gitleaks on your computer.
```

Comment:
If you are suggesting to follow the instructions to install the hook using the pre-commit tool (as opposed to installing the hook manually in the .git directory), you might have forgotten to commit the `.pre-commit-config.yaml` file that pre-commit requires in the root of the repo. Otherwise I'd suggest pointing the link to the more generic [manual installation](https://github.com/gitleaks/gitleaks#pre-commit-hook) section.

Reply:
Of course, you're right. I forgot to add the file into git. Now it should be fixed.